.gitlab/builds.gitlab-ci.yml

```diff
@@ -12,8 +12,9 @@
   script:
     - git clone https://github.com/flux-framework/flux-core
     - cd flux-core
-    - export FLUX_BUILD_DIR=$(pwd)
     - ./autogen.sh
-    - ./configure
-    - make -j 32
-    - cd ..
+    - mkdir build
+    - cd build
+    - ../configure --prefix=$(pwd)
+    - make -j install
+    - export CORE_BUILD_DIR=$(pwd)
```

Comment:
Does this last line do anything? Just curious because if the `script` exits after the last entry in this array, then the `export` is not doing anything here. However, maybe that is not how gitlab works and the export command is special here/

Reply:
This script is never run on its own. It's reused for our `make check` and flux-pmix `make check` using reference tags, so the export is special. See here:
```
        - !reference ['.build-core', 'script']
        - cd $CORE_BUILD_DIR
        - make -j check
```
That `!reference` expands out everything in this script, then runs some more commands, which is why `$CORE_BUILD_DIR` is helpful.

Reply:
Ok, great, thanks for clarifying that!